Sources/SFSafeSymbols/Symbols/SFSymbol+1.1.swift

```diff
@@ -0,0 +1,155 @@
+// Don't touch this manually, this code is generated by the SymbolsGenerator helper tool
+
+// 1.1 Symbols
+@available(iOS 13.1, macOS 11.0, tvOS 13.0, watchOS 6.1, *)
+public extension SFSymbol {
+    /// 􀞸
+    /// Single Localization, 2 Layersets
+    ///
+    /// Layersets:
+    /// - Monochrome
+    /// - Hierarchical (iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0)
+    static let arrowUturnLeftCircleBadgeEllipsis = SFSymbol(rawValue: "arrow.uturn.left.circle.badge.ellipsis")
```

Comment:
Have you tested whether these 1.1 symbols are indeed only available on 13.1 and cause a crash when used on iOS 13? That would mean that the official app gives wrong availability information leading to crashes... If they work on iOS 13, there must be a mistake in the plist that you extracted from the OS.

Reply:
I added some explanation below about why 1.1 exists in this file and not the app one but to answer your question, yes I tried the 1.1 symbols on a 13.0 simulator and it did crash.

Reply:
Thank you! That's very interesting! Bummer we didn't catch that with the CI before, but unfortunately we only test on iOS 13.7, not iOS 13.0